exporters/otlp/otlp_exporter_test.cc

```diff
@@ -0,0 +1,74 @@
+#include "otlp_exporter.h"
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+#include "opentelemetry/proto/collector/trace/v1/trace_service_mock.grpc.pb.h"
+
+#include <gtest/gtest.h>
+
+using namespace testing;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+class OtlpExporterTestPeer : public ::testing::Test
+{
+public:
+  std::unique_ptr<OtlpExporter> GetExporter(
+    proto::collector::trace::v1::TraceService::StubInterface* mock_stub)
```

Comment:
nit: It would be clearer to pass std::unique_ptr<proto::collector::trace::v1::TraceService::StubInterface> here to explicitly express that ownership is transferred.